services/madoc-ts/src/frontend/site/pages/search.tsx

```diff
@@ -1,5 +1,81 @@
 import React from 'react';
+import { SearchFacets } from '../../shared/components/SearchFacets';
+import { SearchResults } from '../../shared/components/SearchResults';
+import { PaginationNumbered } from '../../shared/components/Pagination';
+import { useTranslation } from 'react-i18next';
+import { useLocationQuery } from '../../shared/hooks/use-location-query';
 
-export const Search: React.FC = () => {
-  return <h1>Search</h1>;
+import { createUniversalComponent } from '../../shared/utility/create-universal-component';
+import { UniversalComponent } from '../../../frontend/types';
+
+import { useHistory, useLocation } from 'react-router-dom';
+import { usePaginatedData } from '../../shared/hooks/use-data';
+
+import styled from 'styled-components';
+
+import { SearchResponse } from '../../../types/schemas/search';
```

Comment:
I think I duplicated the types here - throwing an error as the type in `api.ts` is from `types/search`

```
Type 'Promise<import("/opt/build/repo/services/madoc-ts/src/types/search").SearchResponse>' 
is not assignable to 
type 'Promise<import("/opt/build/repo/services/madoc-ts/src/types/schemas/search").SearchResponse>'.
```